xchart/src/main/java/org/knowm/xchart/internal/chartpart/Chart.java

```diff
@@ -141,6 +145,16 @@ public String getYAxisTitle() {
     return yAxisTitle;
   }
 
+  public Color getXAxisTitleColor() {
```

Comment:
I think this stuff belongs in the Styler, not the `Chart` class.

Reply:
already fixed